src/Microsoft.DocAsCode.Utility/StringExtension.cs

```diff
@@ -81,5 +82,15 @@ public static string ToDisplayPath(this string path)
             if (string.IsNullOrEmpty(path)) return null;
             return path.Replace(Path.AltDirectorySeparatorChar, Path.DirectorySeparatorChar);
         }
+
+        public static string GetMd5String(this string content)
+        {
+            using (var ms = new MemoryStream())
+            using (var writer = new StreamWriter(ms))
+            {
+                writer.Write(content);
```

Comment:
no need, it won't write anything if it is null/empty

---
In reply to: [75429946](https://github.com/dotnet/docfx/pull/621#discussion_r75429946) [](ancestors = 75429946)